build/teamcity/cockroach/ci/tests/local_roachtest_fips_impl.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -euox pipefail
+
+fips_enabled=$(cat /proc/sys/crypto/fips_enabled)
+
+if [[ $fips_enabled != "1" ]]; then
+  echo "FIPS mode is not enabled. Exiting."
+  exit 1
```

Comment:
Will this be added as a build config to `Bazel Essential CI`? Should we `exit 0` if FIPS is not enabled? i.e. noop

Reply:
The TeamCity wrapper calls this script only when it exists in the source tree. In other words it's called only when we need to call it.
This particular check happens after we ensure we need to run this test and checks that the underlying OS has proper support for FIPS.

Reply:
right right now I remember you talked about FIPS agents